### PR TITLE
Build & Testing automation

### LINKED ISSUE #13: Build & Testing automation
**Describe the feature you'd like**
As with any project, testing gocui is important. We should set up tools like circle-ci and codecov to automaticly check some things.

**Describe alternatives you've considered**
Testing manually. This is too much work with too many points where it can go wrong.

**Additional context**
/

### PR DESCRIPTION
**Pull request by glvr182**
This PR is here to add a bunch of badges and a circleci config.
I set up a codecov report, a go report badge, and a circleci reviewer already.

Fixes #13

Signed-off-by: Glenn Vriesman <glenn.vriesman@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # GOCUI - Go Console User Interface
-
+[![CircleCI](https://circleci.com/gh/awesome-gocui/gocui/tree/master.svg?style=svg)](https://circleci.com/gh/awesome-gocui/gocui/tree/master)
+[![CodeCov](https://codecov.io/gh/awesome-gocui/gocui/branch/master/graph/badge.svg)](https://codecov.io/gh/awesome-gocui/gocui)
+[![Go Report Card](https://goreportcard.com/badge/github.com/awesome-gocui/gocui)](https://goreportcard.com/report/github.com/awesome-gocui/gocui)
+[![GolangCI](https://golangci.com/badges/github.com/jesseduffield/lazygit.svg)](https://golangci.com/badges/github.com/jesseduffield/lazygit.svg)
```
Comment (mjarkk) [inline README.md]: Why is this is a badge from lazygit?

Comment (glvr182) [reply README.md]: My mistake, will remove it now

**Review by mjarkk — APPROVED** [review]

**Review by mjarkk — APPROVED** [review]

**Comment by glvr182** [comment]
After a bunch of force pushing I managed to get it working @mjarkk @skanehira 
Waiting for your reviews!

**Comment by mjarkk** [comment]
LGTM, I see there are no good tests yet but that's also a bit hard with this project, maybe we can add those later.

**Comment by glvr182** [comment]
We should indeed look into that, we could achieve some basic things with `go test` but i fear that checking if the view was rendered correctly will be a challenge

**Pull request merged**